packages/node/src/internal.ts

```diff
@@ -38,61 +47,45 @@ export type ContextTestFn = (context: object) => boolean
  */
 export type GenerateOptions = Except<
   GeneratePolicyOptions,
-  keyof WritePolicyOptions | 'isAbsolute'
+  keyof WritePolicyOptions
 >
 
 /**
  * Options for `loadCompartmentMap()`
  *
  * @internal
  */
-export type LoadCompartmentMapOptions = Merge<
-  Merge<
-    Omit<BaseLoadCompartmentMapOptions, 'dev'>,
-    Pick<MapNodeModulesOptions, 'conditions'>
-  >,
-  Merge<WithReadPowers, WithPolicyOverride>
+export type LoadCompartmentMapOptions = Simplify<
+  BaseLoadCompartmentMapOptions &
+    Pick<MapNodeModulesOptions, 'conditions'> &
```

Comment:
(old man yells at cloud) The type explicitly saying omit dev where it turns out it was missing tells me typescript doesn't guarantee we'll notice an omission.

Reply:
wait what?  yes, `Omit<T, U>` does not verify that `U` exists in `T`, _but_ `type-fest`'s [Except](https://github.com/sindresorhus/type-fest/blob/main/source/except.d.ts) does. 😄